File: For Fun/Assignment 1/C# version/ShortName.cs
using System;

class ShortName
{
	public static void Main(string[] args)
	{
		Console.Write("First Name: ");
		string fName = Console.ReadLine();
		Console.Write("Last Name: ");
		string lName = Console.ReadLine();
		string sName = fName[0] + ". " + lName.Substring(0, 4);
		Console.WriteLine("Short Name: " + sName);
	}
}

File: For Fun/Assignment 1/Java version/ShortName.java
import java.util.Scanner;

public class ShortName {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("First Name: ");
		String fName = scan.nextLine();
		System.out.print("Last Name: ");
		String lName = scan.nextLine();
		scan.close();
		String sName = fName.charAt(0) + ". " + lName.substring(0, 4);
		System.out.println("Short Name: " + sName);
	}
}
